Classes/Repository/RepositoryIssuesViewController.swift

```diff
@@ -94,6 +100,18 @@ IndicatorInfoProvider {
             }
         }
     }
+    func composeIndicatorTitle(with itemCount: Int) -> String {
```

Comment:
I'm wondering if we should consider moving this out of the `ViewController`'s logic... and write a test for it? 😇